test/Faker/Provider/de_DE/AddressTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Faker\Test\Provider\de_DE;
+
+use Faker\Generator;
+use Faker\Provider\de_DE\Address;
+use Faker\Provider\de_DE\Person;
+
+class AddressTest extends \PHPUnit_Framework_TestCase
+{
+    
+    /**
+     * @var Faker\Generator
+     */
+    private $faker;
+    
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Address($faker));
+        $faker->addProvider(new Person($faker));
+        $this->faker = $faker;
+    }
+    
+    /**
+     * @test
```

Comment:
We don't use the `@test` annotation in the rest of the unit tests, please structure your code accordingly